backend/dataall/modules/shares_base/handlers/ecs_share_handler.py

```diff
@@ -30,6 +30,13 @@ def verify_share(engine, task: Task):
     def reapply_share(engine, task: Task):
         return EcsShareHandler._manage_share(engine, task, SharingService.reapply_share, 'reapply_share')
 
```

Comment:
This worker is not used right? I think we do not need it because the ECS task for persistent emails is scheduled. The worker decorator registers this handler in the Worker Lambda which would trigger the ECS task on demand